### ISSUE TITLE
Any chance it will render radio button group a little better or is it a coding issue?

### ISSUE THREAD

**Issue by haisol** [niklasvh/html2canvas#1175]
Here is the checkbox group on the page:
![orig](https://user-images.githubusercontent.com/1390339/28757926-b10d1cae-754a-11e7-8e83-37c2c3991480.png)

Here is how it looks when sent to canvas img:
![new](https://user-images.githubusercontent.com/1390339/28757939-d6f92ab6-754a-11e7-873a-5d32047749c3.png)

**Comment by niklasvh** [comment]
@haisol What browser/OS are you using? Could you create a jsfiddle with an example so I can try and replicate the issue myself.

**Comment by haisol** [comment]
Sure, I didn't try that, I'll create one tonight after work.

Thanks

**Comment by haisol** [comment]
Any ideas why it would render with the lines through it?  i know I have other libraries I'm using and it's the only control that renders funny...

**Comment by niklasvh** [comment]
I'll have a closer look tonight, but there have been some issues with border radius with percentages as well as radius values that overlap corners. However, I see that you are using 0.4.1 in the examples, which will not be getting any more updates. The library is currently undergoing a rewrite (https://github.com/niklasvh/html2canvas/tree/v1.0.0), so if there is a bug regarding this, it will be fixed there.

**Comment by haisol** [comment]
Think I figured it out...  The code below renders good
 <div class="fb-radio-group form-group field-radio-group-1501639021446">
                <label for="radio-group-1501639021446" class="">Radio Group</label><div class="">
                    <div class="">
                        <label for="radio-group-1501639021446-0">
                            <input name="radio-group-1501639021446" id="radio-group-1501639021446-0" value="option-1" type="radio" checked="checked">Option 1</label></div>
                    <div class="">
                        <label for="radio-group-1501639021446-1">
                            <input name="radio-group-1501639021446" id="radio-group-1501639021446-1" value="option-2" type="radio">Option 2</label></div>
                    <div class="">
                        <label for="radio-group-1501639021446-2">
                            <input name="radio-group-1501639021446" id="radio-group-1501639021446-2" value="option-3" type="radio">Option 3</label></div>
                </div>
            </div>

**Comment by haisol** [comment]
But when I add custom css for a radio group class it renders funky as shown in original post.
```
            <div class="fb-radio-group form-group field-radio-group-1501639021446">
                <label for="radio-group-1501639021446" class="">Radio Group</label><div class="radio-group">
                    <div class="radio">
                        <label for="radio-group-1501639021446-0">
                            <input name="radio-group-1501639021446" id="radio-group-1501639021446-0" value="option-1" type="radio" checked="checked">Option 1</label></div>
                    <div class="radio">
                        <label for="radio-group-1501639021446-1">
                            <input name="radio-group-1501639021446" id="radio-group-1501639021446-1" value="option-2" type="radio">Option 2</label></div>
                    <div class="radio">
                        <label for="radio-group-1501639021446-2">
                            <input name="radio-group-1501639021446" id="radio-group-1501639021446-2" value="option-3" type="radio">Option 3</label></div>
                </div>
            </div>
```

**Comment by niklasvh** [comment]
Could you re-add the jsfiddles you had before your edit?

**Comment by haisol** [comment]
It's the awsome bootstrap checkboxes that causes the render issues:  https://rawgit.com/flatlogic/awesome-bootstrap-checkbox/master/awesome-bootstrap-checkbox.css

**Comment by haisol** [comment]
Jfiddle: http://jsfiddle.net/haisol/bp3z6yg0/

**Comment by haisol** [comment]
Thanks and no problem I will add those back in tonight.

**Comment by haisol** [comment]
The one I already posted is a duplicate of one of the original postings.  Here is the other:
http://jsfiddle.net/haisol/o0sydngs/

**Comment by niklasvh** [comment]
Fixed in 1.0.0 http://jsfiddle.net/gyacgjgw/
